tests/models/nextflow/conftest.py

```diff
@@ -1,39 +1,31 @@
-from typing import Dict, Optional
+from pathlib import Path
 
 import pytest
 
 
-@pytest.fixture(name="nextflow_deliverables")
-def nextflow_deliverables() -> Dict[str, str]:
-    return {
-        "format": "tsv",
-        "id": "CASEID",
-        "path": "PATHTOCASE/fusioncatcher/CASEID.fusioncatcher.fusion-genes.txt",
-        "path_index": "~",
-        "step": "fusioncatcher",
-        "tag": "fusioncatcher",
-    }
-
-
-@pytest.fixture(name="nextflow_deliverables_with_empty_entry")
-def nextflow_deliverables_with_empty_entry() -> Dict[str, Optional[str]]:
-    return {
-        "format": "",
-        "id": "CASEID",
-        "path": "PATHTOCASE/fusioncatcher/CASEID.fusioncatcher.fusion-genes.txt",
-        "path_index": "~",
-        "step": "fusioncatcher",
-        "tag": "fusioncatcher",
-    }
-
-
-@pytest.fixture(name="nextflow_deliverables_with_faulty_entry")
-def nextflow_deliverables_with_faulty_entry() -> Dict[str, str]:
-    return {
-        "faulty_header": "tsv",
-        "id": "CASEID",
-        "path": "PATHTOCASE/fusioncatcher/CASEID.fusioncatcher.fusion-genes.txt",
-        "path_index": "~",
-        "step": "fusioncatcher",
-        "tag": "fusioncatcher",
-    }
+@pytest.fixture(name="deliverable_id")
+def fixture_deliverable_id() -> str:
```

Comment:
```suggestion
def deliverable_id() -> str:
```
Please update all fixtures in this PR to accommodate this. See: https://github.com/Clinical-Genomics/cg/pull/2444

Reply:
I will. Thanks for pointing out

Reply:
done